common/source/docs/common-fettec-onewire.rst

```diff
@@ -1,8 +1,13 @@
-==============
-FETtec OneWire
-==============
+===================
+FETtec OneWire ESCs
+===================
 
-This is a `half duplex serial protocol with a 2Mbit/s Baudrate<https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_FETtec_OneWire>`_ created by Felix Niessen (former Flyduino KISS developer) from FETtec. It supports ESC telemetry, so information from the ESC status is sent back to the autopilot:
+.. note::
+
+   This feature is available in Copter-4.1 (and higher)
+
+OneWire is an `ESC bi-directional communication protocol<https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_FETtec_OneWire>`_ from `FETtec <https://fettec.net>`_.
```

Comment:
having the url twice on a page causes a build error....that is why I removed it from this sentence

Reply:
Why is that a build error ??

Reply:
IDK, but it is...Sphinx is funny about some things